white-van/discussion-board
# [Feat] Add Storybook and ESLint

## [DEV-000] Issue Title

Fixes: #5 


### Changes
Explain the changes made at a higher level
Cleans up Storybook and adds the basic ESLinting rules,

The developers should now have standardized formats and should be able to view and edit Chakra-ui components in Storybook.

### Testing
- Does the code have sufficient automated test coverage? If not, include a reasoning and steps required to manually QA
It doesn't as this is apart of setting up the framework,
To test ESLint,
you must have the ESLint extension installed, or run `npm run lint` on the frontend directory, verify no linting errors occur.

To test storybook,
run `npm install`,
run `npm run storybook`
Verify a basic chakra-ui button shows and is editable + shows the correct theme
Verify the docs plugin works correctly as well as the a11y tab
